packages/vite/src/node/plugins/html.ts

```diff
@@ -799,6 +805,11 @@ export type IndexHtmlTransformHook = (
 export type IndexHtmlTransform =
   | IndexHtmlTransformHook
   | {
+      order?: 'pre' | 'post' | null
+      handler: IndexHtmlTransformHook
+      /**
+       * @deprecated renamed to `order`
+       */
       enforce?: 'pre' | 'post'
       transform: IndexHtmlTransformHook
     }
```

Comment:
```suggestion
export type IndexHtmlTransform =
  | IndexHtmlTransformHook
  | {
      order?: 'pre' | 'post' | null
      /**
       * @deprecated renamed to `order`
       */
      enforce?: 'pre' | 'post'
      /**
       * @deprecated renamed to `handler`
       */
      transform: IndexHtmlTransformHook
    }
  | {
      order?: 'pre' | 'post' | null
      /**
       * @deprecated renamed to `order`
       */
      enforce?: 'pre' | 'post'
      handler: IndexHtmlTransformHook
    }
```
IIUC we want to deprecate `transform`, too?